packages/strapi-plugin-content-manager/admin/src/components/Inputs/index.js

```diff
@@ -24,24 +23,29 @@ import {
 function Inputs({
   allowedFields,
   autoFocus,
+  fieldSchema,
+  formErrors,
   isCreatingEntry,
   keys,
+  labelIcon,
+  metadatas,
   onBlur,
-  formErrors,
   onChange,
   readableFields,
   shouldNotRunValidations,
   queryInfos,
   value,
-  fieldSchema,
-  metadatas,
 }) {
   const {
     strapi: { fieldApi },
   } = useStrapi();
   const { contentType: currentContentTypeLayout } = useContentTypeLayout();
   const { formatMessage } = useIntl();
 
+  const labelIconformatted = labelIcon
+    ? { icon: labelIcon.icon, title: formatMessage(labelIcon.title) }
+    : labelIcon;
```

Comment:
not sure to understand this ternary. If you reach the "else" branch, labelIcon will be undefined. WYT of putting a null / undefined instead?